pkg/runtime/cli.go

```diff
@@ -11,7 +11,6 @@ import (
 	"os"
 	"strconv"
 	"strings"
-
```

Comment:
let's keep one pr one thing please

Reply:
Isn't it a thing now? Is it still necessary to separate the CLI configuration and timeout addition for the app channel?